misc/sync_scripts.py

```diff
@@ -1,50 +1,47 @@
+import sys
 import os
 import ruamel.yaml
 
+SYNC_PROJECT = sys.argv[-1] if sys.argv[-1] else "worker"
```

Comment:
Isn't this always going to be true? even if I would call `python3 sync_scripts.py`, then the last argv will be the name of called script? I guess it doesn't matter because it will exit anyway, it just won't say how the wrong value got assigned into the SYNC_PROJECT var